exercises/conor_finn/ex02_oo_basics/src/main.cpp

```diff
@@ -0,0 +1,53 @@
+#include "circle.h"
+#include "rectangle.h"
+#include "shapeSorter.h"
+#include "square.h"
+#include "triangle.h"
+
+#include <iostream>
+#include <vector>
+
+int main(int, char **) {
+  std::vector<Shape *> shapes;
+
+  // Make some shapes.
+  Rectangle *r1 = new Rectangle(5, 10);
+  Rectangle *r2 = new Rectangle(50, 10);
+  Triangle *t1 = new Triangle(10, 23);
+  Triangle *t2 = new Triangle(2, 60);
+  Square *s1 = new Square(67);
+  Circle *c1 = new Circle(42);
```

Comment:
Using `new` isn't ideal as you then have to delete the pointers at the end (well done for remembering to do this though). You could consider using smart pointers, the standards say to "replace uses of `new` by `std::make_unique`, where possible"